core/src/main/java/org/frankframework/metrics/LocalStatisticsRegistry.java

```diff
@@ -155,13 +155,7 @@ private JsonArrayBuilder getHourlyStatistics(Adapter adapter) {
 		JsonArrayBuilder hourslyStatistics = Json.createArrayBuilder();
 		for (int i=0; i<numOfMessagesStartProcessingByHour.length; i++) {
 			JsonObjectBuilder item = Json.createObjectBuilder();
-			String startTime;
-			if (i<10) {
-				startTime = "0" + i + ":00";
-			} else {
-				startTime = i + ":00";
-			}
-			item.add("time", startTime);
+			item.add("time", String.format("%02d", i));
```

Comment:
Previously, they added also `:00` behind it somehow.. not sure if you still need it.. Just so you know.